tests/sentry/api/endpoints/test_doc_integrations.py

```diff
@@ -0,0 +1,155 @@
+from django.urls import reverse
+from rest_framework import status
+
+from sentry.api.serializers.base import serialize
+from sentry.models.integration import DocIntegration
+from sentry.models.integrationfeature import IntegrationFeature, IntegrationTypes
+from sentry.testutils import APITestCase
+
+
+class DocIntegrationsTest(APITestCase):
+    url = reverse("sentry-api-0-doc-integrations")
```

Comment:
We have this cool thing that you can use which makes the tests a little more concise.
When you set `endpoint` and `method` for and `APITestCase`, you can start using `get_success_response()` and `get_error_response()`:
```py
class DocIntegrationsTest(APITestCase):
    endpoint = "sentry-api-0-doc-integrations"
    ...

class PostDocIntegrationsTest(DocIntegrationsTest):
    method = "post"

    def test_create_invalid_auth(self):
        """Tests that the POST endpoint is only accessible for superusers."""
        self.login_as(user=self.user)
        self.get_error_response(**self.payload, status_code=status.HTTP_403_FORBIDDEN)
```